llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -258,6 +258,12 @@ class MCDCRecordProcessor {
   /// ExecutedTestVectorBitmap.
   MCDCRecord::TestVectors ExecVectors;
 
+  /// Temporary use
```

Comment:
Can you expand on this comment a little?

Reply:
I've modified for `ExecVectorsByCond[0]` as the storage of the final `ExecVectors`.